### ISSUE TITLE
feat: allow re-ordering pages and moving elements among pages.

### ISSUE THREAD

**Issue by batonac** [frappe/print_designer#286]
The new pages are awesome! I'm so excited. These two capabilities would really make it shine.

**Comment by maharshivpatel** [comment]
@batonac moving elements across pages is allowed just press cmd/ctrl. ordering can be good addition

**Comment by batonac** [comment]
Thanks for the tip @maharshivpatel. Ctrl+Click does indeed appear to do the trick, but the moved elements are obviously never registered on the new page. They are both hidden by their z-index initially, and any subsequent moves (after the Ctrl+Click move) jump them back onto the original page:

![Designer Page Elements](https://github.com/frappe/print_designer/assets/4996285/186c269f-bf0a-4523-bc93-3a78e940a951)

**Comment by maharshivpatel** [comment]
@batonac yeah, it's doesn't work sometimes. there is some edge case will fix that. In meantime you can ctrl+a to get them in front of the page and then hold ctrl and move again then they should stay in the page.

**Comment by batonac** [comment]
Unfortunately, the multiple-Ctrl-move trick isn't working for me. As you can see from the `18 in Y` in this screenshot, the element is anchored to the first page, regardless of where I place it on the second page.
![image](https://github.com/frappe/print_designer/assets/4996285/0148b965-fc5c-40e8-aaa9-077d144f2322)

**Comment by maharshivpatel** [comment]
@batonac doesn't happen on my formats 😅. If you don't mind can you share your format. i have faced this same issue in the past but unable to replicate it as of now.

**Comment by maharshivpatel** [comment]
@batonac i think it is because your element is not 100% inside body of page it is on top of header. can you try with smaller element that you can be sure in inside body of second page and not overlapping with header.

**Comment by batonac** [comment]
That could be true @maharshivpatel. Moving elements across pages is working for me as of the latest release.
